custom_scripts/py_scripts/db_scanner/db_scanner.py

```diff
@@ -0,0 +1,255 @@
+        r"(?<!is)[aA]dmin",
+        r"(?<![iI]d)(?<![iI]s)[mM]ail(?![tT]ime)(?![tT]itle)(?![tT]emplate)",
+        r"(?<![fF]ile)(?<![sS]tatus)[uU]ser_*(?![iI]d)(?![lL]evel)(?![lL]ist)(?![tT]ype)",
+        # (?=[nN]ame)* ?? (?<![sS]hipping)
+        r"[aA]ccount(?![nN]um)",
+        r"[pP]assword",  # (?<=[eE]ncrypted)*_*
+        r"(?<![fF]ile)(?<![tT]ask)(?<![rR]oom)(?<![lL]evel)(?<![tT]able)(?<![gG]roup)[nN]ame(?![iI]d)",
+        # (?<=[sS]ur)*(?<=[fF]irst)*(?<=[lL]ast)*(?<=[uU]ser)*
+        r"\[nN]umber",
+        r"(?<![iI])[pP]hone",
+        r"[mM]obile",
+        r"[sS]alt",
+        r"[kK]ey",  # (?<![sS]har)
+        r"(?<!skr)(?<!sk)(?<![tT]r)(?<![dD]escr)(?<![cC]l)(?<![mM]ult)(<?![eE]q)(<?![pP]rinz)(?<![eE]ll)(?<![pP]hil)(?<!h)(?<!v)(?<![tT]ool)[iI]p(?![lL]ine)(?![pP]hthonge)",
+        # (?<=[cC]lient)*_* ?? (?<![zZ])
+        r"cred",  # ?? (?!it)
+        r"[aA]ddr",
+        r"[dD]evice_*(?![iI]d)",
+        r"[nN]ick(?!el)",
+        r"[mM]a[cC]*(?!ge)(?!p)(?!x)(?!rk)(?!ll)(?!na)(?!st)(?!in)(?!t)(?!k)",
+    ]
+    CRITICAL_INFORMATION_VALUES = [
+        {r"'+'*'('*[0-9][\s-]*'('*[0-9]{2,3,4})*[\s-]*[0-9]{}-*[0-9]{}{{10,11}"},
+        {r"[a-zA-Z_ ]*"},
+        {r"[0-9]{1,2,3}'.'[0-9]{1,2,3}'.'[0-9]{1,2,3}'.'[0-9]{1,2,3}"},
+    ]
+    JSON_RESULTS = "results_db.json"
+
+
+def load_hosts(filename: str = DBScannerDefaultValues.JSON_HOSTS) -> list:
+    """
+    Load hosts from the file (useful when it's separated with Grinder)
+    :param filename: filename of file with hosts
+    :return: list of IP-addresses (for this case port is defined)
+    """
+    ip_list = []
+    with open(filename, mode="r") as json_hosts:
+        result_list = load(json_hosts)
+    for value in result_list:
+        if "ip" in value:
+            ip_list.append(value["ip"])
+    return ip_list
+
+
+def save_results(
+    host_dictionary: dict,
+    filename: str = DBScannerDefaultValues.RESULTS_DIRECTORY.joinpath(
+        DBScannerDefaultValues.JSON_RESULTS
+    ),
+) -> None:
+    """
+    Save results from execution to assigned file
+    :param host_dictionary: scanning results
+    :param filename: filename of resulting file
+    :return: None
+    """
+    host_result = dumps(host_dictionary, indent=4)
+    with open(filename, mode="a") as fd:
+        fd.write(host_result + "\n")
+
+
+def find_values_by_key(document: dict or list) -> dict or list or None:
+    """
+    Find different possible critical information by set regular expressions
+    :param document: document from the database
+    :return: found possible critical data
+    """
+    # print(type(document), document)
+    document_list = {}
+    for d in document:
+        # print(f"doc is {d} and type is {type(d)} of value's {type(document[d])}")
+        if not isinstance(document[d], list and dict):
+            for info in DBScannerDefaultValues.CRITICAL_INFORMATION:
+                # print(info, d)
+                regular = re.findall(f"{info}", d)
```

Comment:
Why are you format string here? You don't format anything here.

Maybe we can use
```python3
regular = re.findall(info, d)
```